Marlin/src/core/boards.h

```diff
@@ -308,18 +308,21 @@
 #define BOARD_BTT_SKR_MINI_E3_V1_0    4022  // BigTreeTech SKR Mini E3 (STM32F103RC)
 #define BOARD_BTT_SKR_MINI_E3_V1_2    4023  // BigTreeTech SKR Mini E3 V1.2 (STM32F103RC)
 #define BOARD_BTT_SKR_MINI_E3_V2_0    4024  // BigTreeTech SKR Mini E3 V2.0 (STM32F103RC)
-#define BOARD_BTT_SKR_E3_DIP          4025  // BigTreeTech SKR E3 DIP V1.0 (STM32F103RC / STM32F103RE)
-#define BOARD_JGAURORA_A5S_A1         4026  // JGAurora A5S A1 (STM32F103ZET6)
-#define BOARD_FYSETC_AIO_II           4027  // FYSETC AIO_II
-#define BOARD_FYSETC_CHEETAH          4028  // FYSETC Cheetah
-#define BOARD_FYSETC_CHEETAH_V12      4029  // FYSETC Cheetah V1.2
-#define BOARD_LONGER3D_LK             4030  // Alfawise U20/U20+/U30 (Longer3D LK1/2) / STM32F103VET6
-#define BOARD_CCROBOT_MEEB_3DP        4031  // ccrobot-online.com MEEB_3DP (STM32F103RC)
-#define BOARD_CHITU3D_V5              4032  // Chitu3D TronXY X5SA V5 Board
-#define BOARD_CHITU3D_V6              4033  // Chitu3D TronXY X5SA V5 Board
-#define BOARD_CREALITY_V4             4034  // Creality v4.x (STM32F103RE)
-#define BOARD_CREALITY_V427           4035  // Creality v4.2.7 (STM32F103RE)
-#define BOARD_TRIGORILLA_PRO          4036  // Trigorilla Pro (STM32F103ZET6)
+#define BOARD_BTT_SKR_MINI_MZ_V1_0    4025  // BigTreeTech SKR Mini MZ V1.0 (STM32F103RC)
+#define BOARD_BTT_SKR_E3_DIP          4026  // BigTreeTech SKR E3 DIP V1.0 (STM32F103RC / STM32F103RE)
+#define BOARD_JGAURORA_A5S_A1         4027  // JGAurora A5S A1 (STM32F103ZET6)
+#define BOARD_FYSETC_AIO_II           4028  // FYSETC AIO_II
```

Comment:
Too many lines show up edited here. You probably need to merge or rebase with the latest state of bugfix-2.0.x to resolve this cleanly.